include/AL/alext.h

```diff
@@ -715,16 +715,21 @@ void AL_APIENTRY alGetObjectLabelEXT(ALenum identifier, ALuint name, ALsizei buf
 
 #ifndef ALC_SOFT_system_events
 #define ALC_SOFT_system_events
-#define ALC_PLAYBACK_DEVICE_SOFT                 0x19D4
-#define ALC_CAPTURE_DEVICE_SOFT                  0x19D5
+#define ALC_PLAYBACK_DEVICE_SOFT                   0x19D4
+#define ALC_CAPTURE_DEVICE_SOFT                    0x19D5
 #define ALC_EVENT_TYPE_DEFAULT_DEVICE_CHANGED_SOFT 0x19D6
-#define ALC_EVENT_TYPE_DEVICE_ADDED_SOFT         0x19D7
-#define ALC_EVENT_TYPE_DEVICE_REMOVED_SOFT       0x19D8
+#define ALC_EVENT_TYPE_DEVICE_ADDED_SOFT           0x19D7
+#define ALC_EVENT_TYPE_DEVICE_REMOVED_SOFT         0x19D8
+#define ALC_EVENT_SUPPORTED                        0x19D9
+#define ALC_EVENT_PARTIALLY_SUPPORTED              0x19DA
+#define ALC_EVENT_NOT_SUPPORTED                    0x19DB
```

Comment:
Enum values should begin at column 50 (given the note in `alc.h` and `al.h`), unless the enum name is too long then it's one space after the name. New enums for `ALC_SOFT_` and `AL_SOFT_` extensions should also have the `_SOFT` suffix.

Reply:
Typo corrected here [4aba47c](https://github.com/kcat/openal-soft/pull/938/commits/4aba47cc8f2f03c265843f8758e18df480ef2c0b)!